frontend/src/metabase/actions/utils.unit.spec.ts

```diff
@@ -65,7 +65,7 @@ describe("sortActionParams", () => {
     it("should generate settings for a string field", () => {
       const fields = [createField({ name: "test-field" })];
       const params = [createParameter({ id: "test-field" })];
-      const [id, settings] = getFirstEntry(
+      const [_id, settings] = getFirstEntry(
```

Comment:
with `"destructuredArrayIgnorePattern": "^_"` we'll ignore whatever param in array destructuring that starts with `_` I think that's a good pattern since it's really easy to determine what variable isn't used so we can easily understand it even when e.g. reviewing code on GitHub.